atlasdb-dbkvs/src/main/java/com/palantir/atlasdb/keyvalue/dbkvs/impl/ranges/DbKvsGetRanges.java

```diff
@@ -330,7 +335,14 @@ private String getSimpleRowSelectOneQueryOracle(
     }
 
     private String getPrefixedTableName(TableReference tableRef) {
-        return new PrefixedTableNames(config, new ConnectionSupplier(connectionSupplier)).get(tableRef);
+        switch (dbType) {
```

Comment:
Seems off that ``oracleTableNameGetter`` is assigned in ``Dbkvs`` and ``prefixedTableNames`` here. Can we pass both in the constructor and have only one switch assigning both in ``Dbkvs``.